tests/test_analysis_masks.py

```diff
@@ -55,7 +55,7 @@ def test_merge_masks(masks):
 
```

Comment:
# FIXME test coverage for permutations (nav, sig): (1, 2) (time series); (2, 1) (spectral), (2, 3) (hyperspectral)

This is critical because the whole point of moving to n-dimensional data sets is breaking the (2, 2) assumption. This needs test coverage.

Reply:
And perhaps (1, 1) for good measure. :-)